src/main/java/exception/NoEventOneMonthAgoException.java

```diff
@@ -0,0 +1,16 @@
+package exception;
+
+/*
+* Represents the case when there is no event happened one month ago but the user typed in autoclear
+* */
+public class NoEventOneMonthAgoException extends NuScheduleException {
```

Comment:
This exception seems redundant if we consider auto-clear as an option that can be open or closed. In that sense, you may want to remove this exception? Since having no event happening one month ago is not really a big deal.